.github/file-paths.yaml

```diff
@@ -14,8 +14,8 @@ shared_specs: &shared_specs
 
 frontend_sources: &frontend_sources
   - *shared_sources
-  - "enterprise/frontend/**"
-  - "frontend/!(test)/**"
+  - "enterprise/frontend/!(test)**/**/{!(*.spec),}.{js,jsx,ts,tsx,css}"
+  - "frontend/!(test)**/**/{!(*.spec),}.{js,jsx,ts,tsx,css}"
```

Comment:
tbh I'm not sure about the part `!(test)**`, afaik `**` or globstar is used to include all the subdirectories, usually you put it inside `/` and `/`, but not as a part of another string. For strings afaik you need to use one star `*`